app/src/main/java/org/greenstand/android/TreeTracker/fragments/MapsFragment.kt

```diff
@@ -236,6 +235,9 @@ class MapsFragment : androidx.fragment.app.Fragment(), OnClickListener, OnMapRea
     }
 
     private fun renderTrees() {
+
+        map ?: return
```

Comment:
In what conditions would the map be null? When location is not enabled?

Reply:
While working on this it crashed here due to it being null. This is a quick fix